packages/tools/src/tools/annotation/planarFreehandROITool/closedContourEditLoop.ts

```diff
@@ -225,6 +225,10 @@ function finishEditAndStartNewEdit(evt: EventTypes.InteractionEventType): void {
 
   annotation.data.polyline = worldPoints;
   annotation.data.isOpenContour = false;
+  // If any manual update, triggered on an annotation, then it will be treated as non-autogenerated.
```

Comment:
This bit of detail needs to go into the top level documentation on the contour tool, since it is fairly important to know the behaviour.